draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -1275,8 +1276,8 @@ however, an endpoint MAY change the value of a_rwnd it sends in SACK chunks.</t>
 to create in this association.
 The value of 0 MUST NOT be used.</t>
 <t>A receiver of an INIT chunk with the OS value set to 0 MUST discard the
-packet, SHOULD send a packet in response containing an ABORT chunk and using
-the Initiate Tag as the Verification Tag, and MUST NOT change the state of any
+packet, SHOULD send a packet in response containing an ABORT chunk with the
+T bit set, and MUST NOT change the state of any
```

Comment:
And which Verification Tag should be used?
* If you use 0, the receiver can not verify it. This also violates the rule that only packets containing the an INIT chunk use a Verification Tag of 0.
* If you use the Initiate Tag, the receiver will drop the packet containing the ABORT chunk.

So are you suggesting that no packet containing an ABORT chunk is used, since it does not make a lot of sense to send a packet knowing that the receiver will drop it. It only makes sense for monitoring devices.

Reply:
For a bit more context: this was the result of an observation that we used multiple phrasings to refer to the way the verification tag could be set, and a pretty mindless attempt to normalize on a single phrasing.  This was predicated on an (unverified) assumption that the specification for the respective T flags would be sufficiently precise to cover all cases.  If there's a reason to use a different phrasing here, then we should not make the change.  (Same for below.)

Reply:
Hmm. I wasn't clear enough: If you want the ABORT you are sending to be acceptable for the receiver you MUST NOT set the T-bit and MUST use the initiate tag of the INIT chunk. All other combination do not work. The text you suggested is a technical change, not an editorial cleanup. So if you are not intending a technical change, I would suggest to keep the wording.

I all in favour if improving consistency of text. So if you want to to look at some paragraphs which should be consistent, but you think they are not, please point me to them and I'm more than happy to see if it can be improved.

I explicitly describe

Reply:
Since I think the text is good as it is now, I'm resolving the conversation. If you have a suggestion how to improve consistency, please unresolve the conversation.

Reply:
You have convinced me that my understanding of the details here is incomplete and that the risks of my proposing changes in this space are too large to make that a useful course of action at this stage in the document lifecycle.  Thanks for the extra explanations that demonstrated that my assumptions were flawed.